content/docs/command-reference/stage/add.md

```diff
@@ -0,0 +1,432 @@
+# stage add
+
+Helper command to add or update <abbr>stages</abbr> in `dvc.yaml`. Requires a
+name and a command.
```

Comment:
```suggestion
Helper command to create or update <abbr>stages</abbr> in `dvc.yaml`.
```

Reply:
I'd prefer `add` as that's what the command name is.